ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -0,0 +1,140 @@
+import logging
+
+from selenium.webdriver.common.by import By
+
+from ocs_ci.ocs.ui.views import deployment
+from ocs_ci.ocs.ui.base_ui import BaseUI
+
+
+logger = logging.getLogger(__name__)
+
+
+class DeploymentUI(BaseUI):
+    """
+    Deployment OCS Operator via User Interface
+
+    """
+
+    def __init__(self, driver):
+        super().__init__(driver)
+
+    @property
+    def select_mode(self):
+        return self.mode
+
+    @select_mode.setter
+    def select_mode(self, mode):
+        if not isinstance(mode, str):
+            raise ValueError("mode arg must be a string")
+        self.mode = mode
+
+    @property
+    def select_storage_class(self):
+        return self.storage_class_type
+
+    @select_storage_class.setter
+    def select_storage_class(self, storage_class):
+        if not isinstance(storage_class, str):
+            raise ValueError("storage class arg must be a string")
+        self.storage_class = storage_class
+
+    @property
+    def select_osd_size(self):
+        return self.osd_size
+
+    @select_osd_size.setter
+    def select_osd_size(self, osd_size):
+        if not isinstance(osd_size, str):
+            raise ValueError("osd size arg must be a string")
+        self.osd_size = osd_size
+
+    @property
+    def select_encryption(self):
+        return self.is_encryption
+
+    @select_encryption.setter
+    def select_encryption(self, is_encryption):
+        if not isinstance(is_encryption, bool):
+            raise ValueError("is_encryption arg must be a bool")
+        self.is_encryption = is_encryption
+
+    def install_ocs_opeartor(self):
+        """
+        Install OCS Opeartor
+
+        """
+        logger.info("Click On Operators Tab")
+        self.do_click(deployment["Operators Tab"])
+
```

Comment:
General comment: Make sure the script waits till it sees the right object to work upon. Verify that next UI object is visible before doing any operations on it.

Reply:
@hnallurv I added take_screenshot function on BaseUI class